### ISSUE TITLE
some abbreviation for GPT-2

### ISSUE THREAD

**Issue by super-dinosaur** [super-dinosaur/little-discovery#15]
| 参数 | 全称 | 含义 |
|---|---|---|
| `wte` | Word Token Embedding | token → 向量 |
| `wpe` | Word Position Embedding | 位置 → 向量 |
| `h` | Hidden layers | Transformer block 堆叠 |
| `n_embd` | Number of Embedding dimensions | 隐藏维度（384/768） |
| `n_layer` | Number of Layers | Transformer 层数（12） |
| `n_head` | Number of Heads | 注意力头数（12） |
| `n_ctx` | Number of Context tokens | 最大上下文长度（1024） |
| `embd_pdrop` | Embedding Probability Dropout | dropout 概率 |

**Comment by super-dinosaur** [comment]
`self.h = nn.ModuleList([Block(config.n_ctx, config, scale=True) for _ in range(config.n_layer)])`
造 12 层 transformer，每层做因果自注意力和 MLP，顺序堆起来。
